lib/engines/content_block_manager/lib/tasks/update_version_diffs.rake

```diff
@@ -0,0 +1,17 @@
+namespace :content_block_manager do
+  desc "Update Content Block Manager Version Diffs"
+  task update_version_diffs: :environment do
+    ContentBlockManager::ContentBlock::Version.where(event: "updated", item_type: "ContentBlockManager::ContentBlock::Edition").find_each do |version|
+      edition = version.item
```

Comment:
running this locally I seem to have a version where `item` is `nil` and it fails 🤔

Reply:
wonder how that happened...maybe an Edition got deleted?

Reply:
That's weird, because Versions require an item to be present. Might be something janky in your environment